packages/components/src/styles/link-button-styles.ts

```diff
@@ -100,23 +115,26 @@ export const getLinkButtonStyles = (
       alignItems: 'flex-start',
       justifyContent: 'center',
       width: '100%',
-      minWidth: '54px', // ensure space is already reserved until icon component is loaded (ssr)
-      minHeight: '54px', // ensure space is already reserved until icon component is loaded (ssr)
+      ...(hideLabel && {
+        minWidth: minSizeHideLabel, // ensure space is already reserved until icon component is loaded (ssr)
```

Comment:
maybe we shouldn't place it on the button/link host itself but on the icon host? wouldn't that be much cleaner and results into the same effect?

Reply:
The `p-icon` already has a defined width and height for all sizes. This is because we use `size="inherit"` on the `p-icon`.